diff --git a/cmd/jiq/jiq_test.go b/cmd/jiq/jiq_test.go
index 45f822a..50f6076 100644
--- a/cmd/jiq/jiq_test.go
+++ b/cmd/jiq/jiq_test.go
@@ -41,10 +41,12 @@ func TestjiqRunWithError(t *testing.T) {
 	assert.Equal(0, called)
 }
 
-func (e *EngineMock) Run() jiq.EngineResultInterface {
+type EngineMock struct{ err error }
+
+func (e *EngineMock) Run() *jiq.EngineResult {
 	return &jiq.EngineResult{
-		err:     fmt.Errorf(""),
-		qs:      ".querystring",
-		content: `{"test": "result"}`,
+		Err:     fmt.Errorf(""),
+		Qs:      ".querystring",
+		Content: `{"test": "result"}`,
 	}
 }
